### DISCUSSION TITLE
Why does the font under the barcode become thicker after saving the pdf file

### DISCUSSION THREAD

**Discussion by XMQSVIP** [pymupdf/PyMuPDF#2650] [Looking for help] [answered]
**Why does the font under the barcode become thicker after saving the pdf file? Other fonts are normal**
[1.pdf](https://github.com/pymupdf/PyMuPDF/files/12568236/1.pdf)

pdf = fitz.open(path, garbage=4, deflate=True, clean=True, pretty=True)
pdf.save(path

        
![Qbbbb34](https://github.com/pymupdf/PyMuPDF/assets/87030672/cc89ede2-13eb-4462-a7aa-abab6086acce)

**Comment by JorjMcKie** [comment, marked as answer]
Please provide reproducing data. The link above does not work.
As an immediate remedy: do not use `clean=True` ... even if it sounds so reassuring 😎.

**Reply by XMQSVIP** [reply]
Thank you. Your solution was the right one
